scripts/generate_extensions_function.py

```diff
@@ -26,12 +26,23 @@
 }
 
 functions = {}
-ext_dir = os.path.join('..', '.github', 'config', 'extensions.csv')
-ext_hpp = os.path.join("..", "src","include","duckdb", "main", "extension_entries.hpp")
-reader = csv.reader(open(ext_dir))
-# This skips the first row (i.e., the header) of the CSV file.
-next(reader)
 
+# Parses the extension config files for which extension names there are to be expected
+def parse_extension_configs():
+    ext_configs = [
+        os.path.join('..', '.github', 'config', 'out_of_tree_extensions.cmake'),
+        os.path.join('..', '.github', 'config', 'in_tree_extensions.cmake')
+    ]
+    extension_names = []
+    for file in ext_configs:
+        with open(file, 'r') as file:
+            data = file.read().replace('\n', '')
+        pattern = re.compile(r"duckdb_extension_load\s*\(\s*([A-z\_0-9]*)\s")
```

Comment:
Somehow a nit, but this seems to be only a regex on the cmake file, so stuff like `if (! WIN32)` has no effect (unsure whether this is already taken into account).

Reply:
yea we may want to look into doing this in some other way. I would vote to keep this slightly hacky solution for now, then look into improving it as we need